authors/apps/authentication/social.py

```diff
@@ -0,0 +1,62 @@
+import facebook
+from google.auth.transport import requests
+from google.oauth2 import id_token
+from django.contrib.auth import authenticate
+from .models import User
+
+
+def register_social_user(email, username):
+    user = User.objects.filter(email=email)
+
+    if user.exists():
+        current_user = User.objects.get(email=email)
+        current_user.is_active = True
+        current_user.save()
+        registered_social_user = authenticate(email=email, password="Andelatesting2")
```

Comment:
I notice the password is plain for all to see. Does this mean that as long as I know the email of a user (which is public to everyone) I have access to the user's account once they register using social auth?